### ISSUE TITLE
autophagy of mitochondrion parentage

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#27885]
This is a child of mitochondrion organization, but that seems wrong, it isn't organizing the mitochondrion, it's digestng it.
@pgaudet  are you OK for me to remove this in my next editing session?

**Comment by pgaudet** [comment]
@marcfeuermann

**Comment by marcfeuermann** [comment]
Yes, it is not a child of mitochondrion organization, but indeed a process of degradation of mitochondria.
So it will be fine to remove it.
Thanks a lot.
Marc.

**Comment by pgaudet** [comment]
@ValWood @marcfeuermann just want to confirm this. in GO 'organization' is defined as 'A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, **or disassembly** of an x within a cell. ' see for eg https://amigo.geneontology.org/amigo/term/GO:0006996

I find this confusing but if we say disassembly is NOT organization then we need to change all other similar children. 

Thanks, Pascale

**Comment by marcfeuermann** [comment]
All depends about how you understand disassembly... Is degradation disassembly ??? I'm not sure about this.
In fact the word disassembly is confusing. We should try to avoid such kinds of terms.

**Comment by ValWood** [comment]
I agree, but I don't feel that it is wrong to use assembly when the components are recycled.

We have terms  which definitely fit this:
ribosome disassembly
spliceosome disassembly
nuclear membrane disassembly
mitotic spindle  disassembly
nucleosome diassembly

and many that likely do but I am not sure
clathrin coat disassembly
SNARE complex disassembly
  ESCRT complex disassembly
kinetochore disassembly

some which have no annotations and probably should not have been added
GO:1905692  endoplasmic reticulum disassembly

but overall there are not many disassembly terms

(I think it should exclude degradation)

**Comment by ValWood** [comment]
...largely, it does seem to exclude degradation. The autophagy are an anomaly

**Comment by marcfeuermann** [comment]
Autophagy is indeed considered as a recycling process, but I think that only basal compounds are recycled (lipids, amino acids, ...), not parts of mitochondria or any other organelles. So for autophagy I would not use the term  disassembly.

**Comment by pgaudet** [comment]
cellular component disassembly is defined as A cellular process that results in the breakdown of a cellular component.

Sounds like degradation to me, but maybe we just need to clarify the definition. What would 'disassembly' be, if not degradation?

**Comment by deustp01** [comment]
Maybe it's a pathway boundary issue: a large complex structure is converted into its macromolecular components which might then be re-used (tubulin treadmilling; nuclear lamina disassembly and reassembly connected to mitosis) or might be degraded further. Naive biology question: does anything ever get salvaged out of a lysosome for reuse? If not, then this two-part distinction is irrelevant: as soon as a mitochondrion is targeted for autophagy, it is destined to be fully degraded.

**Comment by pgaudet** [comment]
OK, this is all fine by me. Just make are all terms are treated the same way. I see that 
GO:0097352 autophagosome maturation is_a GO:0032984 protein-containing complex disassembly

and GO:0006914 autophagy is_a [GO:0044248 cellular catabolic process

(I didn't look at the other terms). 

In fact I thought this was intentional for the autophagy refactoring. 

Pascale

**Comment by ValWood** [comment]
ChatGPT:

In cell biology, "disassembly" and "degradation" are processes that occur within cells but refer to different mechanisms and outcomes:

Disassembly in Cell Biology:
Definition: Disassembly in cell biology often refers to the breakdown of cellular structures or organelles into their constituent parts.
Purpose: Disassembly can be a regulated process during cell division (mitosis or meiosis), where cellular structures such as the nuclear envelope, spindle apparatus, and other organelles are dismantled to ensure proper segregation of genetic material.
Example: During mitosis, the nuclear envelope breaks down, and the spindle apparatus disassembles after chromosome segregation to form two daughter cells.
Degradation in Cell Biology:

Definition: Degradation in cell biology refers to the breakdown or removal of cellular components or molecules, often through enzymatic processes.
Purpose: Degradation serves to remove damaged or unnecessary cellular components, regulate cellular processes, and recycle biomolecules for energy or building blocks.
Example: Proteasomal degradation involves the targeted breakdown of proteins tagged for degradation by ubiquitin molecules. Another example is lysosomal degradation, where cellular waste and obsolete organelles are engulfed by lysosomes and broken down by hydrolytic enzymes.
In summary, disassembly in cell biology involves the breakdown of cellular structures during specific cellular processes like division, while degradation refers to the breakdown or removal of cellular components or molecules, typically for maintenance, regulation, or recycling purposes.
Message ChatGPT
ChatGPT can make mistakes. Check important info.

I'm not sure this helps but I think the crux is the macromolecule/building block distinction. Can you reuse the components again in the same pathway, or were they fully degraded?

**Comment by ValWood** [comment]
GO:0097352 autophagosome maturation is_a GO:0032984 protein-containing complex disassembly
seems odd ,but I can see why from the definition:

Removal of PI3P and Atg8/LC3 after the closure of the phagophore and before the fusion with the endosome/lysosome (e.g. mammals and insects) or vacuole (yeast), and that very likely destabilizes other Atg proteins and thus enables their efficient dissociation and recycling. PMID:28077293

( it  depends whether the macromolecules of the autophagosome are reused or targeted for degradation ?)


and GO:0006914 autophagy is_a [GO:0044248 cellular catabolic process
this is definitely OK?

**Comment by ValWood** [comment]
Note that other autophagy terms
pexophagy,
piecemeal autophagy of the nucleus etc
do NOT have corresponding "x organization" or "x dissembly" terms.

Is it OK that I remove these (I am trying to get only the genes involved in "mitochondrion organization" so that we can model the separate mitochondrial organization processes and I'd like to remove autophagy from this set.

This seems consistent with the discussion? The more detailed discussion about what constitutes "assembly" can wait if we agree this is not assembly?

**Comment by ValWood** [comment]
I think we agree that  the autophagy terms should not have organization parent.
I would like to proceed and remove,  and postpone the larger philosophical discussion about  organization ve disassembly vs degradation?

**Comment by ValWood** [comment]
CC @hattrill  we also discussed that this was causing problems.

I will remove "mitochondrion organization" and "organelle disassembly"
as parents 

@pgaudet  it might be useful on an editors call to refine the definition of disassembly (I liked @deustp01  distinction that to be disassembly the components need to be able to be reused in the same process)

**Comment by ValWood** [comment]
Also for reticulophagy and ribophagy
